scripts/exdump_monitor.sh

```diff
@@ -439,7 +439,7 @@ DTIM_latest_amsr2=-1.01
 DTIM_earliest_crsfdb=-1.50
 DTIM_latest_crsfdb=-0.51
 
-$ushscript_dump/bufr_dump_obs.sh $dumptime 0.50 1 satwnd gpsro 1bamua avcspm \
+$USHbufr_dump/bufr_dump_obs.sh $dumptime 0.50 1 satwnd gpsro 1bamua avcspm \
```

Comment:
Revert to using original variable, $ushscript_dump. It is already defined in JOBSPROC_DUMP_MONITOR.